fr-French/Localizable.strings

```diff
@@ -10,42 +10,42 @@
 "create" = "Créer";
 "created" = "Créé";
 "delete" = "Supprimer";
-"done" = "Fini";
+"done" = "Terminé";
 "modify" = "Modifier";
 "retry" = "Réessayer";
 "save" = "Sauvegarder";
 // Left menu
 "alias" = "alias";
 "aliases" = "alias";
-"mailbox" = "mailbox (boîte mail)";
-"mailboxes" = "mailbox (boîtes mail)";
+"mailbox" = "boîte mails";
+"mailboxes" = "boîtes mails";
```

Comment:
`Boîte mail` or `Boîte mails` ? I'm not sure which is better and above all what's right...